perses/app/relative_point_mutation_setup.py

```diff
@@ -209,6 +209,12 @@ def __init__(self,
             complex_positions[:protein_n_atoms, :] = protein_positions
             complex_positions[protein_n_atoms:, :] = ligand_positions
 
+            # Convert positions back to openmm vec3 objects
```

Comment:
Have you had a chance to make an issue about this?

Reply:
I made an issue about this in OpenMM: https://github.com/openmm/openmm/issues/3213